src/pages/SearchPage.js

```diff
@@ -0,0 +1,88 @@
+import PropTypes from 'prop-types'
+import { useState } from 'react'
+import styled from 'styled-components/macro'
+import Button from '../components/Button'
+import CropItem from '../components/CropItem'
+import Header from '../components/Header'
+
+SearchPage.propTypes = {
+  crops: PropTypes.array.isRequired,
+  onClickDetails: PropTypes.func.isRequired,
+  onClickFavorites: PropTypes.func.isRequired,
+}
+
+export default function SearchPage({
+  crops,
+  onClickFavorites,
+  onClickDetails,
+}) {
+  const [searchTerm, setSearchTerm] = useState('')
+  const [searchResults, setSearchResults] = useState([])
+
+  return (
+    <Wrapper>
+      <Header>Search</Header>
+      <Input
+        placeholder="Search your favorite crops  🔍"
+        name="Search"
+        value={searchTerm}
+        onChange={handleChange}
+      />
+      <Output>
+        {searchTerm.length === 0 ? (
+          <>
+            {crops.map(({ id, attributes }) => (
+              <CropItem
+                key={id}
+                name={attributes.name}
+                image={attributes.main_image_path}
+                onClick={() => onClickDetails(id)}
+              />
+            ))}
+          </>
+        ) : searchResults.length === 0 ? (
+          <p>No Crops found</p>
+        ) : (
+          <>
+            {searchResults.map(({ id, attributes }) => (
+              <CropItem
+                key={id}
+                name={attributes.name}
+                image={attributes.main_image_path}
+                onClick={() => onClickDetails(id)}
+              />
+            ))}
+          </>
+        )}
```

Comment:
(change in Line 20 to 
```
const [searchResults, setSearchResults] = useState(crops)
```
)

```suggestion
     {searchResults.length === 0 ? (
          <p>No Crops found</p>
        ) : (
          <>
            {searchResults.map(({ id, attributes }) => (
              <CropItem
                key={id}
                name={attributes.name}
                image={attributes.main_image_path}
                onClick={() => onClickDetails(id)}
              />
            ))}
          </>
        )}
```